packages/xo-server-load-balancer/src/density-plan.js

```diff
@@ -165,7 +165,10 @@ export default class DensityPlan extends Plan {
   // Test if a VM migration on a destination (of a destinations set) is possible.
   _testMigration({ vm, destinations, hostsAverages, vmsAverages }) {
     const {
-      _thresholds: { critical: criticalThreshold },
+      _thresholds: {
+        cpu: { critical: criticalThresholdCpu },
+        memoryFree: { critical: criticalThresholdMemoryFree },
+      },
```

Comment:
The commit message is not easy to understand, maybe use something like: *CPU and memory threshold are separate*

Also, do you know which commit introduced this bug?

Reply:
I don't know, this bug has probably been around for a while, I can try to find out when it was introduced.

Reply:
Likely since the beginning: 5a825bd45